### PR TITLE
Jack rollback

### PR DESCRIPTION
**Pull request by gugakatsi**

### CODE REVIEW

**Comment by gugakatsi** [comment]
@Swati4star there are merge conflicts with FunFunacts module and old loginActivity , i'll fix it few hours probably.

**Comment by gugakatsi** [comment]
i guess is should clsoe PR #14 , already included here , since i pushed to master early accidenly.

**Comment by Swati4star** [comment]
I think you should squash these commits into a single one.
I will also review these, and merge it soon. 👍

File: Android/app/src/main/java/tie/hackathon/travelguide/BusList.java [hunk 1]
```diff
@@ -194,6 +194,7 @@ private void getBuslist() {
                 .url(uri)
                 .build();
         //Setup callback
+        //Setup callback
```
Comment (Swati4star) [inline Android/app/src/main/java/tie/hackathon/travelguide/BusList.java]: repeated line

**Comment by Swati4star** [comment]
Thanks 👍 
I have merged it. 
Fixes https://github.com/Swati4star/Travel-Mate/issues/24

**Pull request merged**